lib/valkyrie/persistence/fedora/query_service.rb

```diff
@@ -126,6 +126,25 @@ def custom_queries
 
     private
 
+      def find_inverse_references_by_unordered(resource:, property:)
+        ensure_persisted(resource)
+        content = content_with_inbound(id: resource.id)
+        property_uri = adapter.schema.predicate_for(property: property, resource: nil)
+        resource_uri = adapter.id_to_uri(resource.id)
+        ids = content.graph.query([nil, property_uri, resource_uri]).map(&:subject).map { |x| x.to_s.gsub(/#.*/, '') }.map { |x| adapter.uri_to_id(x) }
+        ids.uniq! if adapter.standardize_query_result?
+        ids.lazy.map { |id| find_by(id: id) }
+      end
+
+      def find_inverse_references_by_ordered(resource:, property:)
+        ensure_persisted(resource)
+        content = content_with_inbound(id: resource.id)
+        resource_uri = adapter.id_to_uri(resource.id)
```

Comment:
There's a fair bit of duplication between this method and `find_inverse_references_by_unordered` — maybe move the three lines that are the same into `find_inverse_references_by`?

Reply:
Yes, I had all this originally in find_inverse_references_by but rubocop didn't like the complexity of that method. Ensure_persisted could be moved out but if we did the same to the other lines then they'd just have to be passed back in as parameters which seems extraneous as resource and property already are parameters.